setup.py

```diff
@@ -142,7 +142,6 @@ def readfile(fname):
         'Hierarchical Dirichlet Process, HDP, Random Projections, '
         'TFIDF, word2vec',
 
-    license='LGPL',
```

Comment:
Yes, that's the one I meant, that v2+ line in `setup.py`. Hopefully it's OK like this... I'm not gonna hire a lawyer for this :)

We've had exactly one version clarification request in 5 years, so probably not a major issue for most users.